sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -438,3 +438,8 @@
   border: 1px solid $common-border-color !important;
   border-radius: 8px !important;
 }
+
+//Style to add padding to the bullet item.
+.formio-datagrid-unset.student-activity-grid .table-bordered td {
```

Comment:
Is there a way to this be more generic or there is no need to reuse it anywhere else?

Reply:
It is something which can potentially come again. So will try to make it generic.